pkg/appClone/AppCloneService.go

```diff
@@ -594,24 +612,50 @@ func (impl *AppCloneServiceImpl) CreateWf(oldAppId, newAppId int, userId int32,
 			AppWorkflowMappingDto: nil, //first create new mapping then add it
 			UserId:                userId,
 		}
+		thisWf, err = impl.appWorkflowService.CreateAppWorkflow(thisWf)
+		if err != nil {
+			impl.logger.Errorw("error in creating workflow without external-ci", "err", err)
+			return nil, err
+		}
+		impl.logger.Debugw("app workflow created")
```

Comment:
not needed

Reply:
have removed this debug log, didn't need this really